vm/policy.go

```diff
@@ -38,6 +42,54 @@ func PolicyHandling(t string, path string, o PolicyOptions) error {
 		return err
 	}
 
+	client := pb.NewPolicyServiceClient(conn)
+
+	req := pb.Policy{
+		Policy: policyEventData,
+	}
+
+	if resp, err := client.HostPolicy(context.Background(), &req); err == nil {
+		if resp.Status == 1 {
+			fmt.Print("Success")
+		} else {
+			return fmt.Errorf("failed to send policy")
+		}
+	}
+
+	return nil
+}
+
+func sendPolicyOverHttp(policyEventData []byte) error {
+
+	timeout := time.Duration(5 * time.Second)
+	client := http.Client{
+		Timeout: timeout,
+	}
+
+	request, err := http.NewRequest("POST", "http://127.0.0.1:8080/policy", bytes.NewBuffer(policyEventData))
+	request.Header.Set("Content-type", "application/json")
+	if err != nil {
+		return err
+	}
+
+	resp, err := client.Do(request)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+
+	respBody, err := ioutil.ReadAll(resp.Body)
```

Comment:
`ioutil` is deprecated so let's use `io`
```suggestion
	respBody, err := io.ReadAll(resp.Body)
```